checks.d/http_check.py

```diff
@@ -199,12 +201,12 @@ def _load_conf(self, instance):
         ignore_ssl_warning = _is_affirmative(instance.get('ignore_ssl_warning', False))
         skip_proxy = _is_affirmative(instance.get('no_proxy', False))
 
-        return url, username, password, http_response_status_code, timeout, include_content,\
+        return method, data, url, username, password, http_response_status_code, timeout, include_content,\
```

Comment:
can you put `method`, `data` behind `password`? Just to keep some sort of relevance ordering reading from left to right.